### ISSUE TITLE
quantitative: Rule 921170 always flags in error (PL 3 rule)

### ISSUE THREAD

**Issue by RedXanadu** [coreruleset/go-ftw#393]
Rule 921170 is not a detection rule: it is an admin/scaffold rule designed to make the HTTP parameter pollution rules.

```
SecRule ARGS_NAMES "@rx ." \
    "id:921170,\
    phase:2,\
    ⋮
    setvar:'TX.paramcounter_%{MATCHED_VAR_NAME}=+1'"
```

```
$ ./ftw quantitative --crs-path ~/.git/coreruleset --corpus-lang=eng --corpus-source=news --corpus-year=2020 --corpus-size=10K --paranoia-level=4
11:09AM INF ⏳Running quantitative tests with 10 goroutines
Run 10000 payloads in 9.657304833s
Total False positive ratio: 26481/10000 = 2.6481
False positives per rule id:
  920220: 130 false positives
  920221: 130 false positives
  920272: 3299 false positives
  920273: 10000 false positives
  921170: 10000 false positives
  ⋮
```

**Comment by RedXanadu** [comment]
It looks like go-ftw also cannot correctly handle rule 920273 (from the results above).
